### PR TITLE
New package: git-cola-2.6

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/git-cola/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'git-cola'
+pkgname=git-cola
+version=2.6
+revision=1
+build_style=python-module
+hostmakedepends="python"
+depends="git git-perl python-PyQt4"
```
Comment (Vaelatern) [inline srcpkgs/git-cola/template]: `git-perl python-PyQt4`h

File: srcpkgs/git-cola/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="Highly caffeinated Git GUI"
+maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
+license="GPL-2"
+homepage="https://github.com/$pkgname/$pkgname"
```
Comment (Vaelatern) [inline srcpkgs/git-cola/template]: No variables in the `homepage` variable. Makes update check a lot harder to parse (for humans).

**Comment by xtraeme** [comment]
`noarch=yes`

File: srcpkgs/git-cola/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+version=2.6
+revision=1
+build_style=python-module
+noarch=yes
+hostmakedepends="python"
+depends="git-perl python-PyQt4"
+short_desc="Highly caffeinated Git GUI"
+maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
+license="GPL-2"
+homepage="https://github.com/$pkgname/$pkgname"
```
Comment (Vaelatern) [inline srcpkgs/git-cola/template]: `https://github.com/git-cola/git-cola`

Comment (thypon) [inline srcpkgs/git-cola/template]: I like with $pkgname expansion

Comment (Vaelatern) [inline srcpkgs/git-cola/template]: `http://repo.steampowered.com/steamos/pool/main/s/${pkgname}` currently on the update list at `https://repo.voidlinux.eu/void-updates/void-updates.txt`, but okay if you insist.

Comment (Vaelatern) [inline srcpkgs/git-cola/template]: The lines are of identical length :)

**Comment by Vaelatern** [comment]
Looks mergeable to me.

**Pull request merged**